Makefile

```diff
@@ -61,6 +61,7 @@ AUR_RELEASE_OPTIONS ?=
 .PHONY: release
 release: test
 	[ -n "$(VERSION)" ]
+	[ "v$(VERSION)" != "$(shell git tag -l --sort -version:refname | head -n 1)" ]
 	git tag -s -m v$(VERSION) v$(VERSION)
```

Comment:
There should be no reason to explicitly check because `git tag` will fail (with an informative error) if you try to create a tag that already exists. That's even better than your check because it ensures not only that you don't re-release with the latest version again but you don't re-release with _any_ known version again.

Reply:
Ah right, make sense. Will remove this commit.